webpack.config.js

```diff
@@ -1,11 +1,11 @@
-const path    = require("path")
-const webpack = require("webpack")
+const path = require("path");
+const webpack = require("webpack");
```

Comment:
- [ ] Great job on adding the webpack and babel.config files into your project however you stll need to configure the bable loader to run your `jsx `or `js` files. Please add a rule for JSX and/or JS files in your webpack configuration, using the babel-loader and the @babel/preset-react preset. For example:

```js

module.exports = {
  //...
 module: {
    rules: [
      {
        test: /\.(js)$/,
        exclude: /node_modules/,
        use: ['babel-loader'],
      },
    ],
  },

//...
};


```